etk/core.py

```diff
@@ -546,16 +582,28 @@ def _relevant_text_from_context(text_or_tokens, results, field_name):
                             new_end = tokens_len
                         else:
                             new_end = end + 5
+
+                        # result['context']['text'] = ' '.join(
+                        #     text_or_tokens[new_start:new_end])
+                        # result['context']['tokens_left'] = text_or_tokens[
+                        #     new_start:start]
+                        # result['context'][
+                        #     'tokens_right'] = text_or_tokens[end:new_end]
+
                         relevant_text = '<etk \'attribute\' = \'{}\'>{}</etk>'.format(field_name,
                                                                                       ' '.join(text_or_tokens[
                                                                                                start:end]).encode(
                                                                                           'utf-8'))
                         result['context']['text'] = '{} {} {} '.format(
-                            ' '.join(text_or_tokens[new_start:start]).encode('utf-8'),
+                            ' '.join(text_or_tokens[new_start:start]).encode(
+                                'utf-8'),
                             relevant_text,
                             ' '.join(text_or_tokens[end:new_end]).encode('utf-8'))
-                        result['context']['tokens_left'] = text_or_tokens[new_start:start]
-                        result['context']['tokens_right'] = text_or_tokens[end:new_end]
+                        result['context']['tokens_left'] = text_or_tokens[
+                            new_start:start]
+                        result['context'][
+                            'tokens_right'] = text_or_tokens[end:new_end]
+
```

Comment:
changes like these are everywhere, just a rearrangement of code snippets, please revert to the original and add your changes only